.github/ISSUE_TEMPLATE/suggest_wallet.yaml

```diff
@@ -48,13 +48,6 @@ body:
       description: Please provide a URL (e.g. to the website of the wallet).
     validations:
       required: true
-  - type: input
-    id: wallet_live_date
-    attributes:
-      label: When did the wallet go live to users?
-      description: Please provide a date when the wallet was usable by the public
-    validations:
-      required: true
```

Comment:
I would add this back in.

I think we need it given listing policy requires 6 months. While we might not need it in the actual wallet data file, its good information to have in the issues being filed.